ANSWERS.md

```diff
@@ -1,4 +1,15 @@
-- [ ] Why would you use class component over function components (removing hooks from the question)?
-- [ ] Name three lifecycle methods and their purposes.
-- [ ] What is the purpose of a custom hook?
-- [ ] Why is it important to test our apps?
\ No newline at end of file
+- [x] Why would you use class component over function components (removing hooks from the question)?
+
+Class components helps make the same function written in an easier way over function components. 
```

Comment:
*Class components are also important for building upon previous components - If you think back to applied JS --> when we could create a new object based on a previous.